app/controllers/patients_controller.rb

```diff
@@ -450,9 +450,15 @@ def bulk_update
     end
     patients = current_user.get_patients(patient_ids)
 
+    # Slightly more performant to calculate this outside the loop below
+    params_without_monitoring = params.except('monitoring')
+    params_without_monitoring[:diffState] = params_without_monitoring[:diffState].without('monitoring')
+
     patients.each do |patient|
-      update_monitoring_fields(patient, params, non_dependent_patient_ids.include?(patient[:id]) ? :patient : :dependent,
-                               params[:apply_to_household] ? :group : :none)
+      # We never want to update closed records monitoring status via the bulk_update
+      update_params = patient.monitoring ? params : params_without_monitoring
+      update_monitoring_fields(patient, update_params, non_dependent_patient_ids.include?(patient[:id]) ? :patient : :dependent,
+                               update_params[:apply_to_household] ? :group : :none)
```

Comment:
Can a patient controller test case be added to ensure that closed patients will not have their monitoring status updated? This code looks good to me I just want to guard against breaking it in the future.

Reply:
Added!